tasks/download.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Download latest version
+  get_url:
+    url: "https://download.nextcloud.com/server/releases/nextcloud-{{ nextcloud_download_version }}.zip"
+    dest: "{{ global_cache_dir | mandatory }}/"
+  become: false
+  delegate_to: localhost
+
+- name: Extract Nextcloud into tmp dir
+  unarchive:
+    src: "{{ global_cache_dir }}/nextcloud-{{ nextcloud_download_version }}.zip"
+    dest: "/tmp"
+    owner: "{{ nextcloud_system_user }}"
+    group: "{{ nextcloud_system_user }}"
+
+- name: Copy Nextcloud from tmp dir to install dir
```

Comment:
Why not extract it directly to the specified path?

Reply:
because i want to remvoe one directory. when you extract the zip you get /tmp/nextcloud/* and if you want to name your folder something else